src/api/journalEntries/journalEntry.ts

```diff
@@ -101,8 +101,8 @@ export const useCreateJournalEntry = () => {
   const queryClient = useQueryClient();
 
   return useMutation(createJournalEntry, {
-    onSuccess: () => {
-      queryClient.invalidateQueries("journalEntries")
+    onSuccess: async () => {
+      await queryClient.invalidateQueries("journalEntries")
```

Comment:
This is useful so that once a user mutates the journal entries, the cached data is then updated to contain the updated data.